### PR TITLE
Add sgx-detect utility

### PR DESCRIPTION
**Pull request by jethrogb**

### CODE REVIEW

File: sgxs-tools/src/sgx_detect/help.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Common SGX installation issues
```
Comment (jethrogb) [inline sgxs-tools/src/sgx_detect/help.md]: This file should be part of the website, not this repo

File: sgxs-tools/src/sgx_detect/tests/mod.rs [hunk 2]
```diff
@@ -0,0 +1,966 @@
...
+            .build(enclave_loader)?;
+        Ok(())
+    }
+}
+
+define_run_enclave!(RunEnclaveProdWl);
+
+impl RunEnclaveProdWl {
+    fn try_loader<L: Load>(enclave_loader: &mut L) -> Result<(), Error> {
+        let enclave = include_bytes!("/home/jethro/projects/intel-test-enclave/test_enclave.sgxs");
```
Comment (akash-fortanix) [inline sgxs-tools/src/sgx_detect/tests/mod.rs]: Remove local path

Comment (jethrogb) [reply sgxs-tools/src/sgx_detect/tests/mod.rs]: Yup, good catch, thanks.

**Comment by jethrogb** [comment]
Self-reviewing because this needs to land

bors r+

**Pull request merged**